ethereumetl/jobs/export_all_common.py

```diff
@@ -144,8 +144,7 @@ def export_all_common(partitions, output_dir, postgres_connection_string, provid
                     'receipt': insert(RECEIPTS),
                     'token': insert(TOKENS),
                 },
-                converters=[ListJoinItemConverter('topics', ','),
-                            ListJoinItemConverter('function_sighashes', ','),
+                converters=[ListJoinItemConverter('function_sighashes', ','),
                             Numeric38Converter('token_transfer', 'value')]
```

Comment:
Do we still need the Numeric38 converter here?

Reply:
Indeed this make no difference in postgre, I didn't change what was not needed (difficulty attributes in block also work even changing the table type)

Reply:
Removing the converter